api/src/controllers/articles.ts

```diff
@@ -1,73 +1,96 @@
 import { RequestHandler } from "express";
-import * as prismic from "@prismicio/client";
 import asyncHandler from "express-async-handler";
-import {
-  ArticlePrismicDocument,
-  Clients,
-  ContentType,
-  ResultList,
-  Article,
-} from "../types";
+import { Displayable, Clients, ResultList, Article } from "../types";
 import { Config } from "../../config";
-import { transformArticle } from "../transformers/article";
 import { HttpError } from "./error";
-import { articlesContentTypes, graphQueryArticles } from "../helpers/articles";
-
-type PathParams = { contentType: string };
+import {
+  paginationElasticBody,
+  PaginationQueryParameters,
+  paginationResponseGetter,
+} from "./pagination";
 
-// TODO Unused now, but for when we add querying capabilities
 type QueryParams = {
   query?: string;
-  "identifiers.identifierType"?: string;
-};
+} & PaginationQueryParameters;
 
 type ArticlesHandler = RequestHandler<
-  PathParams,
+  never,
   ResultList<Article>,
+  never,
   QueryParams
 >;
+
 const articlesController = (
   clients: Clients,
-  config: Config // TODO Unused now but required when we move to using ElasticSearch
+  config: Config
 ): ArticlesHandler => {
-  const prismicClient = clients.prismic;
+  const index = config.contentsIndex;
 
-  return asyncHandler(async (req, res) => {
-    try {
-      const searchResponse = await prismicClient.get<
-        ArticlePrismicDocument & {
-          contentType: ContentType | ContentType[];
+  const getPaginationResponse = paginationResponseGetter(config.publicRootUrl);
+
+  try {
+    return asyncHandler(async (req, res) => {
+      const queryString = req.query.query;
+
+      const searchResponse = await clients.elastic.search<Displayable<Article>>(
+        {
+          index,
+          body: {
+            ...paginationElasticBody(req.query),
+            _source: ["display"],
+            // TODO This can all be cleaned up later on - it's just to test basics.
+            track_total_hits: true,
+            query: {
+              bool: {
+                should: queryString
+                  ? [
+                      {
+                        match: {
+                          "display.title": {
+                            query: queryString,
+                            analyzer: "whitespace",
+                            operator: "OR",
+                            boost: 1000,
```

Comment:
As you say, to be left for later, but probably worth removing the unused stuff here
```suggestion
                          "display.title": {
                            query: queryString,
```